.travis.yml

```diff
@@ -12,14 +12,16 @@ addons:
 matrix:
   include:
     - os: linux
-      env: CUDA=8.0.61-1
+      env: CUDA=8.0.61-1 CMAKE_OPTIONS="-PENGUINV_BUILD_TEST=ON"
+      before_install: source script/travis/install_cuda.sh
     - os: osx
     - os: linux
       env: STATIC_ANALYSIS=ON
-
-before_install:
-   - source script/travis/before_script.sh
+      before_install: pip install cppclean
```

Comment:
This command fails because of cppclean internal errors in code. Maybe we leave `git clone` as it was then.